invenio_oauthclient/utils.py

```diff
@@ -105,11 +114,7 @@ def oauth_register(form):
     """
     if form.validate():
         data = form.to_dict()
-        if not data.get('password'):
```

Comment:
It is checked here https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/utils.py#L154. The check was there because if there password was None then the [hash_password](https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/utils.py#L155) function was giving an error.